### ISSUE TITLE
关于在ACM3025.mat数据集上的运行结果

### ISSUE THREAD

**Issue by bbjy** [Jhy1993/HAN#2]
您好，非常感谢您的工作，以及将代码开源。
我在运行您的程序的时候得到的结果和您论文中不是很一致，所以想请问您几个问题，谢谢！
1）您论文中得到的结果就是在您现在开源这份的代码的参数下运行的结果吗？按照您论文中，在ACM数据集Training=20%的时候，Micro-F1=89.22， Macro-F1=89.40，而我这边运行的结果是： micro_f1: 0.8626 ;  macro_f1: 0.8637（KNN中k=5）；
2）还想请问您，您在训练的时候有没有出现过拟合现象呢，按照现在这份代码中的情况，在epoch=100的时候early stop ，但这时的training accuracy已经为100%，但是测试的acc却只有86.12%。不知道您是否也遇到这种情况。
3）请问您base_gattn.py `中的micro_f1函数就是您计算最终结果时候使用的函数吗？我不是很理解里面的下面这行代码：
`predicted = tf.round(tf.nn.sigmoid(logits))
按照这种计算方式，在predicted中就会出现对某些节点类别的预测是[1,1,1]情况，那么这个时候不管真实节点类别是什么，这个预测结果都会被加到true positives中？并且打印出来的tp:  664 tn:  2788 fp:  1462 fn:  1461，总和是6375？？而那个数据集里的node number是3025？？

期待您的回复，非常感谢！

**Comment by Jhy1993** [comment]
1、你可以试试将中间epoch结果打印出来，89%左右应该是比较正常的。
2、我也遇到类似的现象。train和val的表现基本一致应该不是过拟合。test会低一些。
3、代码是在gat基础上修改的，我没有用base_gattn.py 中的micro_f1。而是将embedding取出来得到x，y送入到sklearn中的KNN或者Kmean做实验。

**Comment by bbjy** [comment]
感谢回复！其实我得到的结果也是将embedding保存后送入sklearn中的KNN之后得到的结果，但还是和您的有些差距。不过还是很感谢您的工作以及您的解答，谢谢！
